docker-compose.yml

```diff
@@ -6,14 +6,13 @@ services:
     restart: unless-stopped
     ports:
       - ${CORE_P2P_PORT:?err}:${CORE_P2P_PORT:?err} # P2P
+      - 127.0.0.1:${CORE_RPC_PORT:?err}:${CORE_RPC_PORT:?err} #RPC
```

Comment:
Do we want to publish the RPC port always?

Reply:
Yes, I think so. It will be required for `mn status` commands. Advanced users can manually comment this out and start with docker-compose directly.

Reply:
Agree. Then we don’t need to pass aditional params in startCore